hazelcast-client/src/main/java/com/hazelcast/client/proxy/ClientRingbufferProxy.java

```diff
@@ -94,8 +94,9 @@ protected void onInitialize() {
             @Override
             public PortableReadResultSet decodeClientMessage(ClientMessage clientMessage) {
                 final RingbufferReadManyCodec.ResponseParameters params = RingbufferReadManyCodec.decodeResponse(clientMessage);
-                final PortableReadResultSet readResultSet
-                        = new PortableReadResultSet(params.readCount, params.items, params.itemSeqs);
+                final PortableReadResultSet readResultSet = new PortableReadResultSet(
+                        params.readCount, params.items, params.itemSeqs,
+                        params.nextSeqExist ? params.nextSeq : -1);
```

Comment:
maybe naming with an appropriate constant to the context increases readability instead of `-1`

Reply:
Added.